### PR TITLE
feat: add MA account creation styling 

### PR DESCRIPTION
**Pull request by cliu02**
# Pull Request Template

## Issue Overview

This PR addresses #3759.

- [X] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

As part of the Mandated Accounts MVP, this PR applies the updated Figma form styles to the account creation page (#3759).

## How Can This Be Tested/Reviewed?

To review this PR:
1. Go to the `/create-account` page. 
2. Compare the form styling for this page with the [Figma](https://www.figma.com/file/ihaZA5QibRRJFyBdARKn55/User-Accounts?node-id=2591%3A4012&mode=dev) and ensure consistency. 

Note: The date picker will have styling differences as this work will be captured in another ticket. We have also determined that the font, button styling, and duplicate password field will remain the same for now. Please let me know if you notice any other styling differences that should be accounted for in this ticket. 

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: shared-helpers/src/locales/general.json [hunk 1]
```diff
@@ -189,14 +191,17 @@
   "application.name.dobHelper": "For example: 01 19 2000",
   "application.name.emailPrivacy": "We will only use your email address to contact you about your application.",
   "application.name.firstName": "First Name",
+  "application.name.familyName": "Family name",
+  "application.name.givenName": "Given name",
   "application.name.lastName": "Last Name",
   "application.name.middleName": "Middle Name",
-  "application.name.middleNameOptional": "Middle Name (optional)",
+  "application.name.middleNameOptional": "Middle name (optional)",
```
Comment (emilyjablonski) [inline shared-helpers/src/locales/general.json:198]: We can't change any of these strings in the English file without also having translations. It's just proven to be too easy to not know that the non-English files need changes still. If we have the new cased versions, we can update them in all languages, but the strings should stay the same with a note/ticket to update them all if we don't have em yet.

**Comment by emilyjablonski** [comment]
Similar feedback here in terms of pulling in the Seeds Card and not using @apply tailwind styles :)

File: shared-helpers/src/locales/general.json [hunk 2]
```diff
@@ -7,6 +7,8 @@
   "account.application.noApplicationError": "No application with that ID exists",
   "account.application.return": "Return to applications",
   "account.createAccount": "Create Account",
+  "account.createAccount.title": "Create an Account",
```
Comment (emilyjablonski) [inline shared-helpers/src/locales/general.json:10]: Do you mind alphabetizing these?

File: sites/public/src/pages/create-account.tsx [hunk 3]
```diff
@@ -72,182 +73,339 @@ export default () => {
...
+            </Heading>
+            {requestError && (
+              <AlertBox className="" onClose={() => setRequestError(undefined)} type="alert">
+                {requestError}
+              </AlertBox>
+            )}
+            <SiteAlert type="notice" dismissable />
+          </div>
+          <Form id="create-account" onSubmit={handleSubmit(onSubmit)}>
+            <div className="form-card__group border-b space-y-3 mx-12 p-0 py-8">
```
Comment (emilyjablonski) [inline sites/public/src/pages/create-account.tsx:91]: Fully uptaking Card will require removing divs like this and using the Card sections. Reminder that there are examples in the application :)

**Comment by ColinBuyck** [comment]
Going to wait to review here to see the [changes with the account card patterns ](https://github.com/bloom-housing/bloom/pull/3820#issuecomment-1912544442) but I'm here to pair and support any tweaks to make this pattern work for your use-cases as well!

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
Hey T! Two overall comments here:
* We don't need to toggle this work off
* We should not be seeing the Doorway styles in core, so things like the increased font weight on the header or the style changes to the form fields should not be present. Now that we are using seeds components, the Doorway-specific styles are added only in Doorway.

File: sites/public/src/pages/create-account.tsx [hunk 4]
```diff
@@ -72,182 +73,161 @@ export default () => {
...
+                name="email"
+                label={t("application.name.yourEmailAddress")}
+                validation={{ required: true, pattern: emailRegex }}
+                error={errors.email}
+                errorMessage={t("authentication.signIn.loginError")}
+                register={register}
+                controlClassName={styles["create-account-input"]}
+              />
+            </CardSection>
+            <CardSection className="mx-12 p-0 py-8 space-y-3">
```
Comment (emilyjablonski) [inline sites/public/src/pages/create-account.tsx:171]: I wouldn't expect to see any class names for spacing added onto the Card components - we should be able to take whatever is in seeds.

**Comment by emilyjablonski** [comment]
I'm still seeing some Doorway styles - please let me know if you want to do a paired PR review to confirm which styles stay! If you look at the application, you can see the expected styles. The biggest thing I'm seeing is the styling of the inputs is still in the Doorway style. Given the uptake of seeds, I would expect to be in a place where we can delete potentially the entire module file. Doing overrides just on this page would be unexpected.

**Comment by cade-exygy** [comment]
> I'm still seeing some Doorway styles - please let me know if you want to do a paired PR review to confirm which styles stay! If you look at the application, you can see the expected styles. The biggest thing I'm seeing is the styling of the inputs is still in the Doorway style. Given the uptake of seeds, I would expect to be in a place where we can delete potentially the entire module file. Doing overrides just on this page would be unexpected.

Screenshot 2024-01-31 at 3 58 47 PM
Which ones specifically are Doorway styles? It was my understanding that the ones annotated with orange are Doorway specific, while the blue are both.

**Comment by emilyjablonski** [comment]
@cade-exygy Yep yep you're correct, the orange indicators are Doorway and the blue are core. So the core changes are: (1) Left aligned icon, (3) Given name, (4) Family name, (5) Helper text, (6) Month selector, (7) One email input, (8/9) One password input. I think it will be helpful to check out the application for example styles, so for example here: https://bloom.exygy.dev/create-account There are styles added to the inputs that we don't want in core in terms of the border and background.

**Review by emilyjablonski — APPROVED** [review]
Looks great! Thank you for the updates!

**Review by ColinBuyck — APPROVED** [review]
Looks good, just two little design nits: 1) the spacing between name fields is showing as 16 px in the figma but 20 px in the PR and 2) the spacing below Your email address is 12 px in the Figma but totaling to 20px in the PR. Good to merge after those though!

Screenshot 2024-02-07 at 3 30 33 PM

Screenshot 2024-02-07 at 3 26 45 PM

**Pull request merged**